### PR TITLE
Adding a dummy field for display the text style of the biblio.

### PR DESCRIPTION
**Pull request by RoySegall**
biblio settings site install
booklet site install

### CODE REVIEW

File: biblio.module [hunk 1]
```diff
@@ -365,7 +365,26 @@ function biblio_entity_info() {
...
+ * Implements hook_field_extra_fields().
+ */
+function biblio_field_extra_fields() {
+  $return = array();
+  $info = entity_get_info('biblio');
+  foreach (array_keys($info['bundles']) as $bundle) {
+    $return['biblio'][$bundle] = array(
+      'display' => array(
+        'get_text' => array(
+          'label' => t('CiteProc'),
```
Comment (amitaibu) [inline biblio.module]: => Render Biblio

File: biblio.module [hunk 2]
```diff
@@ -365,7 +365,26 @@ function biblio_entity_info() {
...
+/**
+ * Implements hook_field_extra_fields().
+ */
+function biblio_field_extra_fields() {
+  $return = array();
+  $info = entity_get_info('biblio');
+  foreach (array_keys($info['bundles']) as $bundle) {
+    $return['biblio'][$bundle] = array(
+      'display' => array(
+        'get_text' => array(
```
Comment (amitaibu) [inline biblio.module]: biblio_get_text

File: biblio.module [hunk 3]
```diff
@@ -365,7 +365,26 @@ function biblio_entity_info() {
...
+          'label' => t('CiteProc'),
+          'description' => t('Display the citeproc of the biblio.'),
```
Comment (amitaibu) [inline biblio.module]: Change description.

File: modules/biblio_ui/biblio_ui.admin.inc [hunk 4]
```diff
@@ -15,5 +15,17 @@ function biblio_ui_admin_form($form, &$form_state) {
...
   );
 
+  $options = array();
+  foreach (biblio_get_biblio_styles() as $plugin_name => $plugin) {
+    $options[$plugin_name] = $plugin['title'];
+  }
+  $form['biblio_citeproc_default_style'] = array(
+    '#type' => 'select',
+    '#title' => t('CiteProc default style'),
+    '#description' => t('Select the default citeproc you would like to display in the CiteProc field.'),
```
Comment (amitaibu) [inline modules/biblio_ui/biblio_ui.admin.inc]: change desc and title

File: modules/biblio_ui/biblio_ui.admin.inc [hunk 5]
```diff
@@ -15,5 +15,17 @@ function biblio_ui_admin_form($form, &$form_state) {
     '#default_value' => variable_get('biblio_debug'),
   );
 
+  $options = array();
+  foreach (biblio_get_biblio_styles() as $plugin_name => $plugin) {
+    $options[$plugin_name] = $plugin['title'];
+  }
+  $form['biblio_citeproc_default_style'] = array(
```
Comment (amitaibu) [inline modules/biblio_ui/biblio_ui.admin.inc]: citeproc

**Comment by RoySegall** [comment]
citeproc:
citeproc style

bibtext:
bibtext style

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**